tiledb/sm/query/ast/CMakeLists.txt

```diff
@@ -69,7 +69,8 @@ if (TILEDB_TESTS)
 
     target_include_directories(ast_test_support_lib
```

Comment:
This statement should have originally been written as `target_link_libraries(ast_test_support_lib PUBLIC Catch2::Catch2)`

Reply:
this removed along with other unnecessary include targeting directories